akka-docs/src/main/paradox/agents.md

```diff
@@ -1,168 +1,5 @@
-function.
-
-Scala
-:  @@snip [AgentDocSpec.scala](/akka-docs/src/test/scala/docs/agent/AgentDocSpec.scala) { #send }
-
-Java
-:  @@snip [AgentDocTest.java](/akka-docs/src/test/java/jdocs/agent/AgentDocTest.java) { #import-function #send type=java }
-
-You can also dispatch a function to update the internal state but on its own
-thread. This does not use the reactive thread pool and can be used for
-long-running or blocking operations. You do this with the `sendOff`
-method. Dispatches using either `sendOff` or `send` will still be executed
-in order.
-
-Scala
-:  @@snip [AgentDocSpec.scala](/akka-docs/src/test/scala/docs/agent/AgentDocSpec.scala) { #send-off }
-
-Java
-:  @@snip [AgentDocTest.java](/akka-docs/src/test/java/jdocs/agent/AgentDocTest.java) { #import-function #send-off type=java }
-
-All `send` methods also have a corresponding `alter` method that returns a `Future`.
-See @ref:[`Future`s](futures.md) for more information on `Future`s.
-
-Scala
-:  @@snip [AgentDocSpec.scala](/akka-docs/src/test/scala/docs/agent/AgentDocSpec.scala) { #alter #alter-off }
-
-Java
-:  @@snip [AgentDocTest.java](/akka-docs/src/test/java/jdocs/agent/AgentDocTest.java) { #import-future #import-function #alter #alter-off type=java }
-
-@@@ div { .group-scala }
-
-## Awaiting an Agent's value
-
-You can also get a `Future` to the Agents value, that will be completed after the
-currently queued updates have completed:
-
-@@snip [AgentDocSpec.scala](/akka-docs/src/test/scala/docs/agent/AgentDocSpec.scala) { #read-future }
-
-See @ref:[`Future`s](futures.md) for more information on `Future`s.
-
-## Monadic usage
-
-Agents are also monadic, allowing you to compose operations using
-for-comprehensions. In monadic usage, new Agents are created leaving the
-original Agents untouched. So the old values (Agents) are still available
-as-is. They are so-called 'persistent'.
-
-Example of monadic usage:
-
-@@snip [AgentDocSpec.scala](/akka-docs/src/test/scala/docs/agent/AgentDocSpec.scala) { #monadic-example }
-
-@@@
-
-## Configuration
-
-There are several configuration properties for the agents module, please refer
-to the @ref:[reference configuration](general/configuration.md#config-akka-agent).
-
-## Deprecated Transactional Agents
-
-Agents participating in enclosing STM transaction is a deprecated feature in 2.3.
-
-If an Agent is used within an enclosing @java[`Scala STM`] transaction, then it will participate in
-that transaction. If you send to an Agent within a transaction then the dispatch
-to the Agent will be held until that transaction commits, and discarded if the
-transaction is aborted. @scala[Here's an example:]
-
-@@@ div { .group-scala }
-
-@@snip [AgentDocSpec.scala](/akka-docs/src/test/scala/docs/agent/AgentDocSpec.scala) { #transfer-example }
-
-@@@
+The akka-agent module was deprecated in Akka 2.5 and has been removed in
+2.6. If there is interest it may be moved to a separate, community-maintained
+repository.
```

Comment:
I think we should remove the page, it's enough that this is mentioned in migration guide.
Maybe add redirect in .htaccess